src/main/java/com/rackspace/feeds/feedscatalog/ResolveHostFilter.java

```diff
@@ -0,0 +1,138 @@
+package com.rackspace.feeds.feedscatalog;
+
+import com.rackspace.feeds.filter.OutputStreamResponseWrapper;
+import com.rackspace.feeds.filter.ServletOutputStreamWrapper;
+import com.rackspace.feeds.filter.TransformerUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+import java.io.*;
+import java.net.URL;
+import java.util.*;
+
+public class ResolveHostFilter implements Filter {
+
+    static Logger LOG = LoggerFactory.getLogger(ResolveHostFilter.class);
+    static final String resolveHostXslt = "feedscatalog-resolve-host.xsl";
+
+    private TransformerUtils transformer;
+
+    /**
+     * This method is called once when the filter is first loaded.
+     */
+    public void init(FilterConfig filterConfig) throws ServletException {
+        LOG.debug( "initializing ResolveHostFilter" );
+
+        try {
+            //load the resolve host xslt from resource
+            URL xsltResource = Thread.currentThread().getContextClassLoader().getResource(resolveHostXslt);
+            if (xsltResource != null) {
+                transformer = TransformerUtils.getInstanceForXsltAsFile(xsltResource.getPath());
```

Comment:
There is a method in TransformerUtils called TransformerUtils.getXsltResourceAsString(). Does that method not work here?